text/0000-rand-core.md

```diff
@@ -0,0 +1,1099 @@
+- Feature Name: rand-core crate
+- Start Date: 2017-09-07
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Publish a new `rand-core` crate, containing:
+
+*   the `Rng` trait,
+*   possibly a `CryptoRng` trait,
+*   extension traits `SeedFromRng` and `SeedableRng`
+*   helper functions and/or default implementations for `Rng` functions,
+
+*For now*, re-export all the above in the `rand` crate. Also add two things to
+`rand`:
+
+*   the `NewSeeded` trait and its implementation for `SeedFromRng`
```

Comment:
This sentence is slightly confusing. You don’t implement trait for a trait. Was this sentence supposed to read as one of the following?

> * a `trait NewSeeded: SeedFromRng`;
> * `NewSeeded`, a supertrait of `SeedFromRng`.

Reply:
Nope. [See this code.](https://github.com/dhardy/rand/blob/master/src/lib.rs#L300)

Reply:
Then 

> the `NewSeeded` trait and its (blanket) implementation for `SeedFromRng` implementors

would be less ambiguous.